modules/andes-core/broker/src/main/java/org/wso2/andes/tools/utils/MessageTracer.java

```diff
@@ -65,15 +86,7 @@ public static void trace(AndesMessage message, String content) {
 	 * @param content message activity
 	 */
     public static void trace(AndesMessageMetadata metadata, String content) {
-        if(log.isTraceEnabled()) {
-            String messageContent = "Message {";
-            messageContent += " Destination: " + metadata.getDestination();
-            if (metadata.getMessageID() > 0) {
-                messageContent += " , Id: " + metadata.getMessageID();
-            }
-            messageContent += " } ";
-            log.trace(messageContent + content);
-        }
+        trace(metadata.getMessageID(), metadata.getStorageQueueName(), content);
```

Comment:
Why do you use getStorageQueueName() here and getDestination() in the previous method? Shouldn't we use the same thing in both cases?